script.js

```diff
@@ -0,0 +1,66 @@
+const displaySection = document.querySelector('.bookstore');
+const author = document.getElementById('author');
+const title = document.getElementById('title');
+const addBtn = document.getElementById('addBtn');
+
+let availableBooks = [];
+
+const addBook = (e) => {
+  e.preventDefault();
+  const addedBook = {
+    title: title.value,
+    author: author.value,
+  };
+
+  // save to localStorage
+
+  const saveToLocalStorage = (availableBooks) => localStorage.setItem('availableBooks', JSON.stringify(availableBooks));
+
+  // get from localStorage
+
+  const getFromLocalStorage = () => {
+    if (JSON.parse(localStorage.getItem('availableBooks'))) availableBooks = JSON.parse(localStorage.getItem('availableBooks'));
+  };
+
+  const displayItem = () => {
+    getFromLocalStorage();
+    displaySection.innerHTML = '';
+    availableBooks.forEach((availableBook, i) => {
+      displaySection.innerHTML += `<div class="availableBook">
+        <p>${availableBook.title}</p>
+        <p>${availableBook.author}</p></div>`;
+
+      const deleteBook = (i) => {
+        const filterBooks = availableBooks.filter((book) => book !== availableBooks[i]);
+        saveToLocalStorage(filterBooks);
+        displayItem();
+      };
+
+      const removeBtn = document.createElement('button');
+      removeBtn.textContent = 'Remove';
+      removeBtn.classList.add('remove');
+      displaySection.appendChild(removeBtn);
+
+      const deleteBtn = document.querySelector('.remove');
+      deleteBtn.addEventListener('click', () => {
+        deleteBook(i);
+      });
```

Comment:
- [ ] The correct book is not being deleted. I entered in `4` authors and books and can't delete the second, third and fourth books individually. I have to start by deleting the first book, the second, the third, and then finally the fourth book. Kindly fix this behavior,

![image](https://user-images.githubusercontent.com/81932478/191210154-ea47284e-bf23-4e24-ba70-277917519218.png)

- [ ] In the entries above, the remove button for `Title 2`  does not work, The remove button for `Title 1` works but deletes `Title 2` and not `Title 1`, Kindly fix this issue as well.

Reply:
- [ ] @team as the previous review noted, you can only remove items following a certain order, I would suggest you use index to delete a particular book and also use the array filter method 
`const removeBook = (bookID) => {
  // bookID can be any other property, e.g. title or author, in as much it is unique
  books = books.filter(({ id }) => bookID !== id); // This assumes that `bookID` parameter is the book id
};
`